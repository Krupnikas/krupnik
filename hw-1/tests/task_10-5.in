30 113 37.373737
