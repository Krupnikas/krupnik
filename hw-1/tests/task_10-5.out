Char: q
Int: 30
Real: 37.373737
